quark-test.c

```diff
@@ -665,8 +649,9 @@ t_fork_exec_exit(const struct test *t, struct quark_queue_attr *qa)
 	assert(qp->proc_tty_minor != 0);
 #endif
 	/* check strings */
-	assert(!strcmp(qp->comm, program_invocation_short_name));
-	assert(!strcmp(qp->filename, binpath(NULL)));
+	assert(!strcmp(qp->comm, "true"));
+	assert(!strcmp(qp->filename, "/bin/true") ||
+	    !strcmp(qp->filename, "/usr/bin/true"));
```

Comment:
Is this side of the `||` needed?

Reply:
I mean, when you run in the host, outside of initramfs, it can be /usr/bin/true